README.md

```diff
@@ -2,6 +2,105 @@
+
+Example Request:
+> "Alexa, Ask Boston Info where I can park during a snow emergency.
+
+
+#### Find any current Alerts
+This is used to getting any alerts for the day put out by the Boston government. The alerts can be about any of the the following services:
+  * street cleaning
+  * trash and recyclying
+  * city building hours
+  * parking meters
+  * tow alerts
+
+Example Request:
+> "Alexa, ask Boston Info if there are there any alerts."
+
+
+#### Find out the latest Three One One
+This intent will give you the most recent BOS:311 reports. BOS:311 is an app where residents of Boston can report non-emergency issues like potholes or traffic light outages.
+
+Example Request:
+> "Alexa, ask Boston Info what's the most recent three one one?"
+>
+> "Alexa, ask Boston Info to tell me the 4 most recent three one ones"
+
+
+### Find any nearby Nearby Food Trucks
+The gets near-by food trucks for the given or stored Address. It returns the first five food trucks it finds within one mile of the given address as well as the time period that the food truck is expected to be at the location.
+
+Example Requests:
+> "Alexa, ask Boston Info where are the nearest food trucks?"
+>
+> "Alexa, ask Boston Info what's for lunch?"
+
+
+### Find any nearby Crime incidents
+The crime incidents intent will tell you any recently reported crimes in your area.
+
+Example Requests:
+> "Alexa, ask Boston Info what's the crime activity in my area?"
+>
+> "Alexa, ask Boston Info what is the crime near me?"
+
+#### Leave Feedback for us!
+This is used to provide feedback about the skill, including bug reports and suggestions for new intents. You can leave feedback on our skill by telling Alexa directly!
+
+Example Request:
+> "Alexa, I have a suggestion for Boston Info."
+>
+> "Alexa, I'd like to report a bug to Boston Info."
+
+
+## Intents in Development:
+These are intents that are currently in development and which we hope to make available soon. We are also looking for user feedback on what intents would be most helpful or would be most wanted.
+- Get your polling location
+- Get the Hours for City Buildings
+- Find nearby public bathrooms
+- Crime incidents intents
+- Check the sailing weather on the Charles River
+- Find nearby Health Centers
+- Check animal control/missing pet notices near you
+- Get the Commuter Rail schedule for the day
+- Find nearby parks, playgrounds, and other public spaces
+- Get notifications about nearby Public Works and construction projects
+- Find nearby car shares
+- Find nearby food trucks
+- Information about Moving permits
+- Adding yard waste pick-up days to our existing Trash Info Intent
+- An intent that combines the information of our most popular intents into one question/response pattern
+- A separate Brookline Info skill
```

Comment:
I wouldn't list these here since this could easily get out of date as we abandon intents. Is it possible to link to the issue page filtered with the new intent tag?